app/resources/admin.html.template

```diff
@@ -83,45 +83,74 @@
         return false;
       }
       used_languages.push(language_code);
-      languages_to_titles[language_code] = language_title;
     }
 
     // Copy the dropdowns and the titles into the text boxes that will
     // actually be POSTed (language_menu_options and repo_titles).
     $('language_menu_options').value = JSON.stringify(used_languages);
-    $('repo_titles').value = JSON.stringify(languages_to_titles);
-
+    var custom_message_table_ids = [
```

Comment:
Sorry I didn't notice that this variable is multiple functions. We should make it a global const and share it across functions.

var CUSTOM_MESSAGE_TABLE_IDS = [
   ...
];